File: scratch/fix_app.js
const fs = require('fs');
const path = 'c:/Users/linw2/daily-diet/src/App.jsx';

try {
    let content = fs.readFileSync(path, 'utf8');
    const target = "const { getUserInfo, isLoggedIn, refreshLogin } = import('./lib/googleAuth')";
    if (content.includes(target)) {
        content = content.replace("const { getUserInfo, isLoggedIn, refreshLogin } = ", "");
        fs.writeFileSync(path, content, 'utf8');
        console.log("Fix applied successfully via Node.js.");
    } else {
        console.log("Target string not found.");
    }
} catch (err) {
    console.error("Error:", err);
}

File: scratch/fix_app.py
import os

path = 'c:/Users/linw2/daily-diet/src/App.jsx'
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = []
for line in lines:
    if "const { getUserInfo, isLoggedIn, refreshLogin } = import('./lib/googleAuth')" in line:
        new_lines.append(line.replace("const { getUserInfo, isLoggedIn, refreshLogin } = ", ""))
    else:
        new_lines.append(line)

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)
print("Fix applied successfully.")
